app/controllers/schemes_controller.rb

```diff
@@ -9,7 +9,7 @@ class SchemesController < ApplicationController
 
   def index
     redirect_to schemes_organisation_path(current_user.organisation) unless current_user.support?
-    all_schemes = Scheme.all.order("service_name ASC")
+    all_schemes = Scheme.all.order(confirmed: :asc, service_name: :asc)
```

Comment:
```suggestion
    all_schemes = Scheme.order(confirmed: :asc, service_name: :asc)
```

nit: `all` not required.